task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -80,9 +84,11 @@ spec:
           TESTPASSED=false
         fi
         if [ $TESTPASSED == false ]; then
-          HACBS_ERROR_OUTPUT="$(make_result_json -r FAILURE -f $failure_num -s `expr $check_num - $failure_num`)"
+          note="Task $(context.task.name) completes, please check RESULT for the json check result"
```

Comment:
SR: note="Task $(context.task.name) complete: For details, check the JSON result. "

Check where?